cluster/images/etcd/migrate/options.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	supportedEtcdVersions = []string{"3.4.18", "3.5.15"}
+	supportedEtcdVersions = []string{"3.4.18", "3.5.16"}
```

Comment:
Should this be updated to [v3.4.34](https://github.com/etcd-io/etcd/releases/tag/v3.4.34)?

Reply:
/cc @ahrtr @jpbetz

Reply:
> Should this be updated to [v3.4.34](https://github.com/etcd-io/etcd/releases/tag/v3.4.34)?

Yes, I think so.  

- It seems that nobody care about etcd 3.4.x obviously it isn't updated for a long time? 
- It's accepted to address it in a separate PR since this one is solely intended to bump the 3.5.x?

Reply:
> It seems that nobody care about etcd 3.4.x obviously it isn't updated for a long time?
A quick check, we didn't build 3.4.18 even. The latest 3.4.x in `kubernetes/k8s.io` is `3.4.13-3`.
https://github.com/kubernetes/k8s.io/blob/ca7a917e59f19fbdab73b07440bf7d843e61126d/registry.k8s.io/images/k8s-staging-etcd/images.yaml#L6
```
    "sha256:528ff5e3c80ca0d58d1ab15dc329dc300f9c3fefaffa4c822ca2c389ce254644": ["3.4.13-3"]
```

> It's accepted to address it in a separate PR since this one is solely intended to bump the 3.5.x?

Agree.